wmi.go

```diff
@@ -1,25 +1,25 @@
-/*
-Package wmi provides a WQL interface for WMI on Windows.
-
-Example code to print names of running processes:
-
-	type Win32_Process struct {
-		Name string
-	}
-
-	func main() {
-		var dst []Win32_Process
-		q := wmi.CreateQuery(&dst, "")
-		err := wmi.Query(q, &dst)
-		if err != nil {
-			log.Fatal(err)
-		}
-		for i, v := range dst {
-			println(i, v.Name)
-		}
-	}
+// Package wmi provides a WQL interface for WMI on Windows.
```

Comment:
Without the comment change the build directive isn't processed
unfortunately.


On 27 Jan 2015, at 08:18, Matt Jibson <notifications@github.com> wrote:

In wmi.go <https://github.com/StackExchange/wmi/pull/7#discussion_r23592541>
:

> -	type Win32_Process struct {
> -		Name string
> -	}
> -
> -	func main() {
> -		var dst []Win32_Process
> -		q := wmi.CreateQuery(&dst, "")
> -		err := wmi.Query(q, &dst)
> -		if err != nil {
> -			log.Fatal(err)
> -		}
> -		for i, v := range dst {
> -			println(i, v.Name)
> -		}
> -	}
> +// Package wmi provides a WQL interface for WMI on Windows.

Why the change in comment style?

—
Reply to this email directly or view it on GitHub
<https://github.com/StackExchange/wmi/pull/7/files#r23592541>.